### ISSUE TITLE
Getting val_loss=nan always

### ISSUE THREAD

**Issue by elcronos** [qqwweee/keras-yolo3#79]
Hey guys so just to clarify in your case, after how many epochs you start getting a loss different than nan?

I first tried and stopped after 5 epochs. Then I changed to 100. I just wanted to know how many epochs you will start getting some results !

<img width="1539" alt="screen shot 2018-06-01 at 2 08 37 pm" src="https://user-images.githubusercontent.com/9023043/40827513-598304e2-65b0-11e8-9dea-6c3f2f569b73.png">

**Comment by qqwweee** [comment]
Your code is outdated. Update it first. See the commit b7cbd24.

**Comment by sxyxf66** [comment]
@qqwweee @elcronos do you fix it? I have the same problem. And  the code is new

**Comment by datduonguva** [comment]
Here is my solution, hope it would help
https://github.com/qqwweee/keras-yolo3/issues/171#issuecomment-522287613
